server/public/styles.css

```diff
@@ -155,6 +165,36 @@ div > h3 {
   animation-iteration-count: infinite;
 }
 
+.big-container,
+.home {
+  font-weight: bolder;
+  font-family: sans-serif;
+  background-color: lightcyan;
+}
+
+.selected-hotel {
+  width: 200px;
+}
+
+.gradient {
+  background-image: linear-gradient(#f8a170, #ffcd61);
+}
+
+.bottom-nav-abs {
+  position: fixed;
+  bottom: 0 !important;
+  width: 100%;
+  height: 15vh;
+  border-radius: 0;
+  background-image: linear-gradient(#f8a170, #ffcd61);
+  z-index: 999;
+}
+
+.filler {
+  height: 30vh;
```

Comment:
I don't recommend `vh` for height, just use `px` if you can. What is this for exactly? The CSS class name is hard to guess.